quinn/src/connection.rs

```diff
@@ -518,6 +518,25 @@ impl Connection {
             .clone()
     }
 
+    /// Wait for the connection to be closed without keeping a strong reference to the connection.
```

Comment:
```suggestion
    /// Wait for the connection to be closed without keeping a strong reference to the connection
```

Quinn doc conventions are not rust doc conventions :upside_down_face: